doc/release-notes.md

```diff
@@ -65,6 +65,14 @@ Notable changes
 P2P and network changes
 -----------------------
 
+- The mempool now tracks transactions submitted via the wallet or rpcs as
+  unbroadcast transactions. Every 10-15 minutes, the node will try to announce
+  unbroadcast transactions until a peer requests it via a `GETDATA`, or the
+  transaction is removed from the mempool for other reasons. If you have a
+  custom setup to use P2P messages to submit a local transaction to your node
```

Comment:
```suggestion
  custom setup to use P2P messages to submit a transaction that belongs to a loaded wallet to your node
```

local means wallet, right?

Reply:
kind of.. I've been using the language "local" because it could be wallet, or submitted via `sendrawtransaction`. similarly, if we're talking about these edge cases, someone could theoretically form a transaction & give it to me out of band, then I submit from one computer to my main node via my highly specific custom P2P setup that involves being disconnected from the greater internet. but I guess every valid transaction involves some loaded wallet? 

lol. it all seems like quite an edge case. maybe here I can just remove the word "local".

new:
> If you have a custom setup to use P2P messages to submit a transaction to your node while it is not connected to other nodes, these changes reduce your initial broadcast guarantees. (#18038)

Reply:
> If you have a custom setup to use P2P messages to submit a transaction to your node while it is not connected to other nodes, these changes reduce your initial broadcast guarantees

This has always been the case. Your pull didn't change that.

Reply:
oh. great point. lol ok I'll update to wallet.

Reply:
is there a reason you've suggested "that belongs to a loaded wallet" rather than just wallet transaction?

Reply:
updated to "wallet transaction", lmk if you have further thoughts

Reply:
how would bitcoin core know that this is a wallet transaction when the wallet is not loaded? Maybe this is clear to everyone reading the sentence, but I think the distinction should be made to avoid potential confusion.

Reply:
ah, this was my misunderstanding. I was interpreting "loaded wallet" to mean <<wallet has spendable funds aka utxos>> vs <<node has wallet component loaded>>

also looked in the code and realized that watchonly is a wallet only thing. 

I get what you're saying now. I'm thinking about how to clarify the language.

Reply:
ok. recruited harding & revamped the description. @MarcoFalke what do you think now? 

@harding thanks so much for your help. I continued tinkering with the language, let me know if you have any further recommendations